src/cogent3/__init__.py

```diff
@@ -323,7 +332,7 @@ def load_seq(
     name, seq = data[0]
     name = label_to_name(name) if label_to_name else name
 
-    result = make_seq(seq, name, moltype=moltype, new_type=new_type)
+    result = make_seq(seq, name, moltype=moltype, new_type=new_type, **kw)
```

Comment:
should you be passing `annotation_offset` to `make_seq` here?

Reply:
Good catch!